src/test/java/org/broadinstitute/hellbender/utils/HistogramUnitTest.java

```diff
@@ -51,6 +51,15 @@ public void testMedianOfEvens() throws Exception {
         Assert.assertEquals(bimodalHist.median(), 15.0, EPSILON, "");
```

Comment:
Patch `testMedianOfEvens()` so that it would have failed with the previous implementation (but don't decrease the total number of values added).

Reply:
done